### DISCUSSION TITLE
Migrating Netbox to new server

### DISCUSSION THREAD

**Discussion by dejantep** [netbox-community/netbox#7979] [Q&A] [answered]
I'v just installed new netbox instance (latest) on new server by following installation guide from the docs and it went smooth.
After that i'v dumped database from my old installation (v2.11.12) and imported it to my new instance also by following guide in the docs.

Everything is working but there are some issues.

1.  After import i could not login into my new instance. My guess is that my account got modified somehow. Could it be that i had same username for new superuser account as for account imported? Solution was to create new superuser account, login and sort it out.
2. I dont see any webhooks, custom fields or custom fields definitions in the admin section after import. They are present in the GUI.
Is this expected behavior?

**Comment by candlerb** [comment, marked as answer]
> I'v just installed new netbox instance (latest) on new server by following installation guide from the docs and it went smooth. After that i'v dumped database from my old installation (v2.11.12) and imported it to my new instance also by following guide in the docs.
> 
> Everything is working but there are some issues.
> 
> 1. After import i could not login into my new instance. My guess is that my account got modified somehow. Could it be that i had same username for new superuser account as for account imported? Solution was to create new superuser account, login and sort it out.

That seems unlikely, because:
1. when you import the database you drop the old database and recreate it from the dump (so any existing admin account is lost)
2. if there were any sort of conflict, you'd have seen it when importing the tables into postgres

Now that you have a new superuser account: can you see the other user accounts you created?  Do you just have to change the passwords?

Are you only using local authentication, nothing fancy like LDAP or REMOTE_AUTH?  Do you have anything unusual in your environment, like FIPS compliance (which might disable MD5?)

> 2. I dont see any webhooks, custom fields or custom fields definitions in the admin section after import. They are present in the GUI.
>    Is this expected behavior?

Yes, in Netbox 3.0 a number of sections have been removed from the Django admin area, and are replaced by new sections in the main navigation on the left.  The Django admin area now has only Groups, Users, Job Results, Permissions, Tokens, Background Tasks, and Installed plugins.

**Comment by dejantep** [comment]
When i think about it now, it has to be mistake on my side meaning i supplied wrong credentials when trying to log in. Confusion might have been caused by the step when user creates new superuser account during fresh install BUT drops that same database when following replication guide. Only local authentication, nothing fancy.

I'v found webhooks, custom fields and such under menu "Other" :)
